documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -263,130 +216,99 @@ public void run() {
 we read Strings from the stream and use the standard Java String Serializer to
-convert the bytes read from the stream into String objects.  Finally, the
-ReaderConfig is created, but at the moment, there are no configuration items
-associated with a Reader, so the empty ReaderConfig is just a place holder as
-Pravega evolves to include configuration items on Readers.
-
-Note that you cannot create the same Reader multiple times.  Basically overtime
-you call createReader() it tries to add the Reader to the ReaderGroup.  If the
-ReaderGroup already contains a Reader with that name, an exception is thrown.
-
-Now that we have an EventStreamReader created, we can start using it to read
-Events from the stream.  This is done on line 26.  The readNextEvent() operation
-returns the next Event available on the Stream, or if there is no such Event,
-blocks for a specified timeout period.  If, after the timeout period has expired
-and no Event is available for reading, null is returned. That is why there is a
-null check on line 27 (to avoid printing out a spurious "null" event message to
-the console).  It is also used as the termination of the loop on line 34.  Note
-that the Event itself is wrapped in an EventRead object.
-
-It is worth noting that readNextEvent() may throw an exception (handled in lines
-30-33).  This exception would be handled in cases where the Readers in the
-ReaderGroup need to be reset to a checkpoint or the ReaderGroup itself has been
-altered and the set of Streams being read has therefore been changed.
-
-So that's it.  The simple HelloWorldReader loops, reading Events from a Stream
+convert the bytes read from the stream into String objects. Finally, we use a `ReaderConfig` object with default values.
+
+Note that you cannot create the same Reader multiple times. That is, an application may call `createReader()` to add new Readers to the Reader Group. But if the Reader Group already contains a Reader with that name, an exception is thrown.
+
+After creating an `EventStreamReader`, we can use it to read
+Events from the Stream. The `readNextEvent()` operation returns the next Event available on the Stream, or if there is no such Event, blocks for a specified time. After the expiry of the timeout period, if no Event is available for reading, then _Null_ is returned. The null check (`EventRead<String> event = null`) is used to avoid printing out a spurious _Null_ event message to the console and also used to terminate the loop. Note that the Event itself is wrapped in an `EventRead` object.
+
+It is worth noting that `readNextEvent()` may throw an exception and this exception would be handled in cases where the Readers in the Reader Group need to reset to a Checkpoint or the Reader Group itself has been altered and the set of Streams being read has been therefore changed.
```

Comment:
This could be better phrased. You should mention the name of the exception type.

Reply:
@tkaitchuck @shrids Expection :Reader time out...?

Reply:
@tkaitchuck Accepted suggestion